Bodabi/Bodabi/Views/Input/Holiday/HolidayInputViewController.swift

```diff
@@ -127,6 +128,25 @@ class HolidayInputViewController: UIViewController {
         isDeleting = state
     }
     
+    private func isUniqueName() -> Bool {
+        
+        guard let holiday = selectedHoliday, let relation = selectedRelation else { return false }
+        
+        var isUnique: Bool = true
+        let request: NSFetchRequest<Holiday> = Holiday.fetchRequest()
+        let currentName: String = relation + "의 " + holiday
+        if let fetchResult = try? databaseManager.viewContext.fetch(request) {
```

Comment:
모두 fetch해온 다음 순회하며 필터링하는 것보다 predicate을 이용하는 것이 메모리 측면에서 이로울 것 같다고 생각합니다
```suggestion
        let predicate = NSPredicate(format:"title = %@", currentName)
        request.predicate = predicate
        if let fetchResult = try? databaseManager.viewContext.fetch(request) {
            if let _ = fetchResult.first {
                isUnique = false
            }
        }
```

Reply:
반영하였습니다!! 좋은 리뷰 감사합니다